package.json

```diff
@@ -152,6 +152,7 @@
 		"@babel/core": "7.4.0",
 		"@babel/register": "7.4.0",
 		"@wordpress/element": "2.3.0",
+		"@wordpress/i18n": "3.3.0",
```

Comment:
How come do we need this here?

Reply:
Good question! I should've elaborated that in PR description. :-)

In regular build of extensions we "externalize" (i.e. don't bundle them into our JS bundles) all `@wordpress/*` dependencies since they're available in core WordPress's global `window`.

In Jest we don't have `window` nor those dependencies available, so we need to include them here as the method I'm testing depends on it. Since it's needed only for tests, I'm adding it under `devDependencies` instead of `dependencies`.

For the same reason there's `@wordpress/element` as it's used by some Tiled gallery snapshot tests.